.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Please see the repo readme for directions on how to make PRs on this repo.
+
+Checklist:
+
+- [ ] if you have added a CDT, it appears in the `cdt_slugs.yaml` file
+- [ ] if you have changed the CDT generator code (`rpm.py`), you have bumped
+  the build number in `conda_build_config.yaml` and have remade all of the
+  recipes via running `python gen_cdt_recipes.py`
+- [ ] if you have added a custom CDT recipe, you have added the name of the CDT
+  with `custom: true` in the `cdt_slugs.yaml` file.
+- [ ] all CDT recipes have build number set by `{{ cdt_build_number }}`
+- [ ] old-style/legacy CDTs have a build string of `multi_sysroot_h{{ PKG_HASH }}_{{ cdt_build_number }}`
+- [ ] new-style CDTs have a build string of `single_sysroot_h{{ PKG_HASH }}_{{ cdt_build_number }}`
```

Comment:
Shall we just do build number? A small number like `10` for old-style. A large number like `50` for new-style

Reply:
That would work too. I thought this might be more robust since it has the pkg hash too and I will be using the build string to see if the cdt needs to be built. That might be overkill for sure.

Reply:
done.